src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -1363,14 +1364,15 @@ private static void executeAdventureOnce(
 
       if (KoLCharacter.hasEquipped(ItemPool.get(ItemPool.DRUNKULA_WINEGLASS, 1))) {
         // The wine glass allows you to adventure while falling down drunk
-      } else if (KoLCharacter.getLimitmode() == Limitmode.SPELUNKY) {
+      } else if (Objects.equals(KoLCharacter.getLimitmode(), Limitmode.SPELUNKY)) {
         // You're allowed to Spelunk even while falling down drunk
-      } else if (KoLCharacter.getLimitmode() == Limitmode.BATMAN) {
+      } else if (Objects.equals(KoLCharacter.getLimitmode(), Limitmode.BATMAN)) {
```

Comment:
Again. Using Objects.equals just obfuscates the code, to my eye.

Reply:
Agreed. A limitmode enum refactor sounds nice for the future.